app/presenters/projects_presenter.rb

```diff
@@ -1,8 +1,9 @@
 class ProjectsPresenter
-  VIEWS = %w[detail dashboard]
-
   def initialize(params)
     @params = params
+    @released_projects = released_projects()
+    @unreleased_projects = unreleased_projects()
+    @detailed_projects = detailed_projects()
```

Comment:
Rubocop (whom I intentionally never invited to this repo) would complain about the unnecessary `()` in these lines. More generally, I'm not sure why these 3 instance variables are being set at all. I don't see references to them, so they might just be duplicating work.

Reply:
These are used in the controller here: https://github.com/artsy/horizon/pull/120/files/21ea2acc6c3d123e0ff4c6ed907072621b322404#diff-757cba42ec9599bac53d4cc33cdde3bfR6

I was planning to add rubocop, my Ruby syntax is pretty rough so it seems like it would help me out -- I wonder if there was a specific reason you excluded it?

Reply:
I believe the controller is calling the methods directly, not using the instance variables.

Re. rubocop, it's probably fine to add now that more people are contributing to this repo. I agree with 95% of what it does but the 5% occasionally grates on me.

Reply:
Yes you're right! Will remove this.